FogCreek/Glitch-Community
# Jumbled clipper: Accessibility Contrast Audit

## Links
* Remix link
https://jumbled-clipper.glitch.me/

* Manuscript link
https://glitch.manuscript.com/f/cases/3328620/Team-URL-field-doesn-t-have-sufficient-contrast
https://glitch.manuscript.com/f/cases/3328635/Error-message-display-on-inputs-doesn-t-have-sufficient-contrast
https://glitch.manuscript.com/f/cases/3328623/Analytics-section-on-team-page-has-insufficient-contrast
https://glitch.manuscript.com/f/cases/3328623/Analytics-section-on-team-page-has-insufficient-contrast

## GIF/Screenshots:
(Updates on left)
![image](https://user-images.githubusercontent.com/519336/59124810-acac8880-892e-11e9-8b1a-9f707676d3da.png)
![image](https://user-images.githubusercontent.com/519336/59124818-b0d8a600-892e-11e9-967a-08b8885e95f4.png)
![image](https://user-images.githubusercontent.com/519336/59124825-b6ce8700-892e-11e9-988f-5c6ee3f21d37.png)
![image](https://user-images.githubusercontent.com/519336/59124848-c2ba4900-892e-11e9-9301-25245084e5a8.png)

## Changes:
* Updated the following colors to improve contrast: `text-on-secondary-background` `error` `notification`
* Changed note on bottom of team analytics section to `tertiary` font color (from `secondary`)

## How To Test:
https://jumbled-clipper.glitch.me/@community
* When logged in, view analytics on team page to see changes in notification color
* When logged in, view note on bottom of analytics
* When logged out, view team handle in profileInfo container to see changes to `text-on-secondary-background`
* When logged out, try reporting abuse to see changes in error color

Note: in terms of focus hovering on our default cover image background, I'm going to handle that in a separate case by replacing our default cover image.
